src/bundler.ts

```diff
@@ -147,33 +183,69 @@ class Bundler {
     }
     inlined.add(resolved);
     const backingDocument: ScannedDocument =
-        await analyzer._analyzeResolved(resolved);
+        await analyzer._scanResolved(resolved);
     const documentAst = dom5.parseFragment(backingDocument.document.contents);
     this.pathResolver.resolvePaths(documentAst, resolved, documentUrl);
-    const importParent = htmlImport.parentNode;
+    let importParent: ASTNode;
+    if (matchers.afterHiddenDiv(htmlImport)) {
```

Comment:
Can you add some comments to explain the significance of checking of the import is before or after a hidden div? Maybe break this out into a `getImportParent` method too.